src/hdusd/mx_nodes/nodes/__init__.py

```diff
@@ -17,25 +17,22 @@
 
 from ...utils import HDUSD_LIBS_DIR
 from .. import log
-from .base_node import create_node_types, MxNodeSocket, MxNode_Output
-from . import categories
+from . import base_node, categories
 
 
-nodedef_types, node_types = create_node_types([
+NodeDef_classes, MxNode_classes = base_node.create_node_types([
```

Comment:
"node_def_classes" and "mx_node_classes" would fit the variables naming convention better.
After all, the capitalized names are used for classes.

Reply:
+1